src/client/src/ui/spotTile/components/spotTile.stories.tsx

```diff
@@ -124,70 +141,79 @@ const onNotificationDismissedClick = action('Notification dismissed')
 
 stories.add('Tile', () => (
   <Story>
-    <div
-      style={{
-        width: '320px',
-        height: '150px'
-      }}
-    >
-      <SpotTile currencyPair={currencyPair} spotTileData={spotTileData} executeTrade={executeTrade} />
-    </div>
+    <Centered>
+      <div
+        style={{
+          width: '320px',
```

Comment:
Assume I add a bunch of comments regarding `rem`s — but no need to change, they're not critical here.